src/components/backstage/plugins/Sidebar.js

```diff
@@ -54,19 +67,6 @@ const MaintainersList = ({ npmData, npmDataLoadingState }) => {
         <circle cx="35" cy="125" r="35" />
       </ContentLoader>
     );
-  } else {
-    inner = (
-      <>
-        <ul className="grid grid-cols-4 gap-3 pb-1">
-          {map(npmData.maintainers, ({ name, email }) => (
-            <Maintainer name={name} email={email} key={email} />
-          ))}
-        </ul>
-        <p className="italic text-gray-400">
-          {npmData.maintainersHelpText}
```

Comment:
I just inverted the order of the conditional here in order to get the loading spinner on the page earlier.